packages/manager/apps/pci-savings-plan/src/hooks/useFilteredConsumption.tsx

```diff
@@ -20,30 +20,21 @@ export const useFilteredConsumption = (locale: string) => {
     month,
   });
 
-  const setFilterByPlan = (consumptionFlavor: SavingsPlanFlavorConsumption) => {
-    const defaultFlavor = consumptionFlavor.flavor;
-    setFlavor(defaultFlavor);
-  };
-
-  useEffect(() => {
-    if (consumption && consumption.flavors?.length > 0) {
-      setFilterByPlan(getBigestConsumption(consumption.flavors));
-    }
-  }, [consumption]);
-
-  const newFlavors = [
-    ...new Set(consumption?.flavors?.map((f) => f.flavor.toLowerCase())),
-  ];
+  const flavorsSize =
+    consumption?.flavors
+      ?.map((f) => ({
+        flavor: f.flavor.toLowerCase(),
+        totalActivePlans: getTotalActivePlans(f),
+      }))
+      .sort((a, b) => b.totalActivePlans - a.totalActivePlans) ?? [];
 
   const flavorOptions = useMemo(() => {
-    return newFlavors.length
-      ? newFlavors
-          ?.map((f) => ({
-            label: f,
-            value: f,
-            prefix: isInstanceFlavor(f) ? 'Instance' : 'Rancher',
-          }))
-          .sort((a, b) => a.label.localeCompare(b.label))
+    return flavorsSize.length
+      ? flavorsSize?.map((f) => ({
+          label: f.flavor,
+          value: f.flavor,
+          prefix: isInstanceFlavor(f.flavor) ? 'Instance' : 'Rancher',
+        }))
       : [];
```

Comment:
```suggestion
    return flavorsSize.map((f) => ({
          label: f.flavor,
          value: f.flavor,
          prefix: isInstanceFlavor(f.flavor) ? 'Instance' : 'Rancher',
        }));
```
This will output the same result since map isn't call for an empty list and flavorsSize cannot be null or you would have an error line 32